### ISSUE TITLE
Cannot build tdebase

### ISSUE THREAD

**Issue by michael-manley** [michael-manley/Trinity_ArchLinux_PKGBUILD#36]
Error Information:
```
[ 80%] Building C object tdm/kfrontend/CMakeFiles/tdm_greet.dir/tdm_greet.c.o
cc1: warning: command line option ‘-fvisibility-inlines-hidden’ is valid for C++/ObjC++ but not for C
In file included from /tmp/trinity/tde-core/tde-tdebase/src/tdebase/tdm/kfrontend/tdm_greet.c:28:
/tmp/trinity/tde-core/tde-tdebase/src/tdebase/tdm/kfrontend/tdm_greet.h:30:10: fatal error: config.ci: No such file or directory
 #include <config.ci> /* for the HAVE_VTS define */
          ^~~~~~~~~~~
compilation terminated.
make[2]: *** [tdm/kfrontend/CMakeFiles/tdm_greet.dir/build.make:103: tdm/kfrontend/CMakeFiles/tdm_greet.dir/tdm_greet.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:27292: tdm/kfrontend/CMakeFiles/tdm_greet.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

**Comment by SlavekB** [comment]
What version do you build?

For the upcoming R14.0.6, this issue should be resolved – see commit [18e56531](https://mirror.git.trinitydesktop.org/gitea/TDE/tdebase/commit/18e5653193ee42955d6bdcd22ce5cffc89f1af40) and [3270212e](https://mirror.git.trinitydesktop.org/gitea/TDE/tdebase/commit/3270212e83752ea85bccd77f56fd5e4b2922e74d).

**Comment by michael-manley** [comment]
Looks like the data from those commits helped fix the bug. I am now able to build tdebase. Thank you.
